tests/core/usage_statistics/test_usage_stats_schema.py

```diff
@@ -362,6 +382,10 @@ def test_usage_stats_expectation_suite_messages():
             ).validate(message["event_payload"])
 
 
+@pytest.mark.unit
+@pytest.mark.timeout(
+    6.0  # this test can take longer than the default timeout, try to reduce it
```

Comment:
yikes. Things like `build` and `clean` scream `filesystem` to me.

```
        "cli.docs.build",
        "cli.docs.clean",
```

Reply:
I looked more and it is just using those strings but not running corresponding commands. This test actually takes significantly less than 1s and I'm removing the slow marker.